script/Ciel/Day 8/08_01_CIEL08_2.txt

```diff
@@ -26,99 +27,113 @@ Senpai wasn't overdoing it. she rebuked Noel-sensei in a genuinely scathing tone
 -- Page 160, Offset 18678. Glued.
 -- 。
-// Took me a long time to translate that line
 .
 }
 [sha:7fd93f7c70229da32cf880d20dc0a883b9f5649d]{
 -- Page 160, Offset 18679.
 -- 時刻は午前11時前。この分なら４時限目には滑り込めそうだ。
-It's just before 11 o'clock. At this rate, it seems that I'll make it just in time for the fourth period.
+It's just before 11 o'clock.
+ At this rate, I should be able to make it back in time for fourth period.
 }
 [sha:347c6c1ec294695412726c17683a03f8959ae1d0]{
 -- Page 161, Offset 18680. Mods: @n.
 -- 「あら、志貴クンもう授業に戻るの？
-"Oh my, will you be returning to class, Shiki-kun?
+"Huh? Are you going back to class already, Shiki-kun?
 }
 [sha:f1069c360aab88442f032b33c380a7e0d74906ef]{
 -- Page 161, Offset 18681. Glued.
 -- 放課後まで休んでいくと思ったのに」
-I thought you'd be resting till the end of the day."
+ I assumed you'd spend the remainder of the day resting."
 }
 [sha:096a96789dca26f0897b63d7882a6d2404b20cbc]{
 -- Page 161, Offset 18682.
 -- 「そうしたいところですけど、体が動くなら授業にでますよ。いつ倒れるか分からないんだし、あんまり休めません」
-"I'd like to do that, but I'll go to class if my body allows it. And I don't know when I'll collapse next, so I can't really rest."
+"Well, I'd love to, but if I'm well enough to move I'm well enough to go to class.
+ There's no way of knowing when I'll collapse next, so I can't let myself get complacent."
 }
 [sha:1f707ff0dddc348ccc93c26bc404fd2a38976c6d]{
 -- Page 162, Offset 18683.
 -- 「ふーん。可愛い顔なのに真面目なんだ。ますますキュンときちゃうかも。
-"Ohh? You've got a cute face, but you're also so earnest. I may be really falling for you.
+"Ohh?
+ You're pretty earnest behind that cute face of yours.
+ I might really be falling for you.
 }
 [sha:22d25686437cd18872e9c1d1de47e7ade01f34e2]{
 -- Page 162, Offset 18684.
 -- 先生ね、無理してるクセに平気そうにしている人って昔からタイプなんだ。あ、もちろん男の人限定で。女の場合はまた別の話だから」
-You know, I've always had a thing for people who try to look fine even though they're pushing themselves too hard. Ah, I'm talking about men only of course. It's a different story if it's a woman."
+You know, I've always had a thing for people who try to put on a brave face even when they're pushing themselves.
+ Ah, I'm only talking about men here of course.
+ It's a totally different story if they're a woman."
 }
 [sha:412ab8a341c29ef259a337fc7fcf0848a65a655b]{
 -- Page 163, Offset 18685.
 -- なんて言いつつ、先生は空いているベッドに腰をかけた。
-While saying that, Sensei sits down on an empty bed.
+As she says this, Sensei takes a seat on one of the empty beds.
 }
 [sha:865df84dbcc6881cac9ef724e977e50e2d718e4d]{
 -- Page 163, Offset 18686.
 -- 悪ふざけで誘っている……のではなく、本当にベッドに横になるつもりらしい。
-It seems that she's not trying to tempt me as a joke, she really is going to lie down on the bed.
+I thought that this was more of her twisted sense of humor,
+ but it seems like she's seriously just planning on having a lie down.
 }
 [sha:4c47ec3d46b0ab8e37c995d7cab611938221e40b]{
 -- Page 164, Offset 18687.
 -- 「じゃ、善良な学生生活、しっかりねー♪
-"Well then, enjoy your virtuous student lifeー♪
+"Well then, enjoy your virtuous student life ♪
```

Comment:
I feel like virtuous doesn't quite capture the nuance here.
It's like the phrasing of "A _decent_ member of society" more so.

Reply:
i feel like virtue is right but maybe "student life of virtue"?